aws/src/main/java/org/apache/iceberg/aws/AssumeRoleAwsClientFactory.java

```diff
@@ -104,4 +110,11 @@ public void initialize(Map<String, String> properties) {
   private String genSessionName() {
     return String.format("iceberg-aws-%s", UUID.randomUUID());
   }
+
+  private static Set<Tag> toTags(Map<String, String> properties) {
```

Comment:
We have a similar method in `AwsProperties`. Can we re-use that?

https://github.com/apache/iceberg/blob/d2bb30f863d7276097c39754b9501ba27403b73c/aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java#L459-L464

Reply:
I think we cannot reuse, this is STS tag, not S3 tag, different class.